src/ploigos_step_runner/step_implementers/create_container_image/buildah.py

```diff
@@ -100,6 +102,35 @@ def _required_config_or_result_keys():
         """
         return REQUIRED_CONFIG_OR_PREVIOUS_STEP_RESULT_ARTIFACT_KEYS
 
+    def get_file_hash(self, application_name, service_name, tar_file):
```

Comment:
this function needs a specfici unit test(s) in https://github.com/ploigos/ploigos-step-runner/blob/main/tests/step_implementers/create_container_image/test_buildah_create_container_image.py and then in the `_run_step` tests this function should be mocked.

example: https://github.com/ploigos/ploigos-step-runner/blob/main/tests/step_implementers/sign_container_image/test_podman_sign.py#L105-L107